assistant/assistant.py

```diff
@@ -251,7 +267,29 @@ async def publish_reliably(queue_message: str) -> None:
 
 
 def get_language(user_prompt) -> str:
-    detected_language = detect(user_prompt)
+    try:
+        # Use langdetect to get an initial guess
+        detected_language = detect(user_prompt)
+    except LangDetectException:
+        return "en"
+
+    # Double check to make sure not en
+    if detected_language != "en":
```

Comment:
@wayangalihpratama Let's only do this for a french detection. Swahili should not prompt a second check.